config/default/simple_sitemap.settings.yml

```diff
@@ -0,0 +1,20 @@
+_core:
+  default_config_hash: 2CVQWCMHHIqT_kLCVL_IRfqt8HP9IrnehZNbhXk2jZ0
+max_links: 2000
+cron_generate: true
+cron_generate_interval: 0
+generate_duration: 10000
+entities_per_queue_item: 50
+remove_duplicates: true
+skip_untranslated: true
+xsl: true
+base_url: 'https://beta.intranet.essex.gov.uk'
```

Comment:
I'm wondering if it's possible for this specific config file to not have to create separate files for each environment for `config_split` and instead put `simple_sitemap.settings:base_url` in config ignore? Genuine question :)

Reply:
I hadn't seen that before, thanks!